contrib/minijinja-cli/template.py

```diff
@@ -0,0 +1,25 @@
+pkgname = "minijinja-cli"
+pkgver = "1.0.15"
+pkgrel = 0
+build_wrksrc = "minijinja-cli"
+build_style = "cargo"
+# cargo install rebuilds without these args in install to not ship feature at runtime
+make_build_args = ["--features=completions"]
+make_build_env = {
+    "ASSET_OUT_DIR": "assets",
+}
+hostmakedepends = ["cargo", "pkgconf"]
+makedepends = ["rust-std", "zstd-devel"]
+pkgdesc = "CLI tool for minijinja, a minimal dependency jinja implementation"
```

Comment:
```suggestion
pkgdesc = "Minimal jinja implementation"
```

best i can come up with but maybe you have even better ideas

Reply:
fine with me, and no, don't have better ideas